tests/tools/oecertdump/host/host.cpp

```diff
@@ -81,97 +89,144 @@ oe_result_t enclave_identity_verifier(oe_identity_t* identity, void* arg)
 void output_certificate(const uint8_t* data, size_t data_len)
 {
 #if defined(__linux__)
-    if (log_file)
+    printf("\n");
+    X509* x509;
+    BIO* input = BIO_new_mem_buf(data, (int)data_len);
+    x509 = d2i_X509_bio(input, NULL);
+    if (x509)
     {
-        fprintf(log_file, "\n");
-        X509* x509;
-        BIO* input = BIO_new_mem_buf(data, (int)data_len);
-        x509 = d2i_X509_bio(input, NULL);
-        if (x509)
-        {
-            X509_print_ex_fp(
-                log_file, x509, XN_FLAG_COMPAT, XN_FLAG_SEP_CPLUS_SPC);
-        }
-        BIO_free_all(input);
-        fprintf(log_file, "\n");
+        X509_print_ex_fp(stdout, x509, XN_FLAG_COMPAT, XN_FLAG_SEP_CPLUS_SPC);
     }
+    BIO_free_all(input);
+    printf("\n");
 #endif
     OE_UNUSED(data);
     OE_UNUSED(data_len);
 }
 
-void validate_certificate(uint8_t* cert, size_t cert_size)
+oe_result_t validate_certificate(uint8_t* cert, size_t cert_size)
 {
     oe_result_t result;
 
     result = oe_verify_attestation_certificate(
         cert, cert_size, enclave_identity_verifier, NULL);
 
     log("Certificate verification result: %s\n", oe_result_str(result));
+
+    return result;
 }
 
-static oe_result_t _gen_cert(oe_enclave_t* enclave)
+oe_result_t get_report_from_cert(
```

Comment:
This function name is sgx specific.  It should have _sgx_ in the name.

Reply:
Renamed and moved to sgx_quote.cpp since it's SGX specific.

---
In reply to: [427455435](https://github.com/openenclave/openenclave/pull/3055#discussion_r427455435) [](ancestors = 427455435)